### PR TITLE
Aip 8105

### PR DESCRIPTION
**Pull request by anaim**

### CODE REVIEW

File: src/schema/object/object.repository.ts [hunk 1]
```diff
@@ -320,13 +321,13 @@ export class ObjectRepository implements IObjectRepository {
 							name: `${FileWriterService.GetSupportSystemNameFromType(type)} ${idx + 1}`,
 						},
 					});
-					const geography: Point = {
-						type: 'Point',
-						coordinates: transformRDToWGS([X, Y]),
+					const geographyRD: GeographyRD = {
```
Comment (fortunato) [inline src/schema/object/object.repository.ts:324]: I imagined we would stick to the GeoJSON format for this column. My thinking was it is best to always stick to a standard for describing a piece of data if it is available.
(See: https://geojson.org/)

File: src/schema/span-installation/dto/base-support-system.input.ts [hunk 2]
```diff
@@ -40,6 +41,10 @@ export class BaseSupportSystemInput {
 	@Field(() => Point, { nullable: true })
 	geography?: PointType;
 
+	@IsOptional()
```
Comment (fortunato) [inline src/schema/span-installation/dto/base-support-system.input.ts:44]: Ideally we'd still make use of the `PointType` (see other comment regarding GeoJSON)

File: src/schema/span-installation/junction-box.repository.spec.ts [hunk 3]
```diff
@@ -90,7 +91,7 @@ describe('Span Installation / JunctionBox / Repository', () => {
 	test('updateJunctionBox()', async () => {
 		prismaServiceMock.spanJunctionBoxes.update.mockResolvedValue(domainJunctionBox);
 		prismaServiceMock.$queryRaw.mockResolvedValue([{ geography: JSON.stringify(junctionBox1.geography) }]);
-		const spy = jest.spyOn(repo, 'getGeographyAsGeoJSON').mockResolvedValue(updateJunctionBoxInput.geography);
+		const spy1 = jest.spyOn(repo, 'getGeographyAsGeoJSON').mockResolvedValue(updateJunctionBoxInput.geography);
```
Comment (fortunato) [inline src/schema/span-installation/junction-box.repository.spec.ts:94]: I think you can do better with naming than this :P

**Review by fortunato — CHANGES_REQUESTED** [review]
I'd happily approve this PR if I wasn't of the opinion that we probably want to use GeoJSON even for the JSON column storing RD coordinates.
What do you think?

File: src/schema/span-installation/dto/base-support-system.input.ts (hunk 2 above)
Comment (anaim) [reply src/schema/span-installation/dto/base-support-system.input.ts:43]: Done

File: src/schema/object/object.repository.ts (hunk 1 above)
Comment (anaim) [reply src/schema/object/object.repository.ts]: Done

File: src/schema/span-installation/junction-box.repository.spec.ts (hunk 3 above)
Comment (anaim) [reply src/schema/span-installation/junction-box.repository.spec.ts]: Done

**Review by dlnr — APPROVED** [review]
This looks good to me, i can work with this to adjust the forms and gps component.

**Pull request merged**